web/libs/editor/src/components/Node/Node.tsx

```diff
@@ -80,6 +81,12 @@ const NodeViews = {
     getContent: (node) => <span style={{ color: "#5a5a5a" }}>from {node.sequence[0]?.frame} frame</span>,
   }),
 
+  VideoClassificationModel: NodeView({
+    name: "Video",
+    icon: (...args: any) => <LsChevron {...args} style={{ rotate: "180deg" }} />,
```

Comment:
I like this idea of using the icon and rotating to direction we want but I want to call out that between LSO and LSE we have different ways to accomplish the same thing and even within LSO we have this method and `FaCheveronUp`, down, left and right.
maybe this should be a thing we look at more closely as part of an FE guild initiative as its definitely out of scope for this ticket